.circleci/config.yml

```diff
@@ -17,11 +17,20 @@ commands:
       versions:
         type: string
     steps:
+      - run:
+          name: Install CMake and native library dependencies
+          command: |
+            sudo apt-get update && sudo apt-get install -yq cmake libicu-dev
+      - run:
+          name: Download Android SQLite
+          command: |
+            git clone https://android.googlesource.com/platform/external/sqlite $HOME/sqlite -b android11-release
```

Comment:
We don't run circleci now, do we need to update downloading logic with submodule for it?

Reply:
This file has been removed from this PR, thanks for catching this.